src/Tag/index.scss

```diff
@@ -0,0 +1,30 @@
+.nds-tag {
+  padding: 8px;
```

Comment:
you could do one of two things here:

use the var:
```css
padding: var(--space-s);
```

or apply the padding via a helper class

```jsx
{/* padding--all--s applies `s` padding (8px) to all sides */}
<div className={`nds-typography nds-tag nds-tag--${kind} padding--all--s`}>
```